README.md

```diff
@@ -89,6 +89,18 @@ When the installations succeed, try to refresh the project and see that it build
 The manual Operation-system specific JDK installation
 is well explained in the [official docs](https://docs.oracle.com/en/java/javase/20/install/overview-jdk-installation.html)
 
+### Building the doc
+
+The current active shell JDK version must be compatible with JDK17 or higher for Antora to build successfully.
+So, just ensure that you have installed JDK 21, as previously described and make it as the current one. 
```

Comment:
```suggestion
So, just ensure that you have installed JDK 21, as described above and make it as the current one. 
```

Reply:
fixed in https://github.com/reactor/reactor-core/pull/3806/commits/8aff270d7287936d0607136c0ca2c12a8c21fff2